server/lib/schema/index.ts

```diff
@@ -2094,6 +2094,64 @@ const schema = new GraphQLSchema({
           },
         ),
       },
+      updateTreeExpertAssignments: {
+        type: GraphQLBoolean,
+        args: {
+          treeId: { type: GraphQLString },
+          honestEmails: { type: GraphQLList(GraphQLString) },
+          maliciousEmails: { type: GraphQLList(GraphQLString) },
+        },
+        resolve: requireAdmin(
+          "You must be logged in as an admin to update experiment expert assignments",
```

Comment:
"update experiment expert assignments" to something like "update the experts for a tree"